app/resources/v1/saphyr/container_material_resource.rb

```diff
@@ -0,0 +1,70 @@
+# frozen_string_literal: true
+
```

Comment:
Again Holy crap!!! I didn't realise you would have to do this. There is a lot of complexity to make this work which almost negates the point of polymorphism. We definitely need to move Saphyr over to GraphQL. Hopefully this is an interim measure. Again could you get Harry to look at this as well.

Reply:
Yeah I wasn't proud of this, but I don't see another way while keeping the API the same.  I assume your comment is to say you like JSON:API even less now, rather than to ask me to change anything here?

Reply:
I think me and JSON:API are just about to split up. I don't see the point in putting any more effort into something that is out of the door in a few weeks.